lib/money-rails/configuration.rb

```diff
@@ -35,7 +35,7 @@ def register_currency=(currency_options)
     end
 
     def set_currency_column_for_default_currency!
-      iso_code = default_currency.iso_code
+      iso_code = default_currency&.iso_code
```

Comment:
warning: the safe navigation operator has been introduced only in Ruby 2.3, but there is not a strict requirement on the ruby version in `gemspec`, so this could break Ruby 2.2 users

Reply:
Thanks. Should we add it to the gemspec, or would you prefer for me to rewrite this for compatibility?

Reply:
Not a maintainer, I can only share a couple of thouhgts.

Since the recent changes in money gem (RubyMoney/money#1087), I guess that this gem could drop compatibility with legacy Ruby/Rails versions, but that should be part of a different PR.

About this PR, since the safe navigation operator is not used in other parts of the code, I would rewrite it

Reply:
Updated